### PR TITLE
adding iOS XCFramework support

### PR DESCRIPTION
**Pull request by spoorthipujariadobe**
Adding XCFramework support for iOS by removing bundled iOS libraries. ACPCore and AEPAssurance XCFrameworks need to be downloaded and added to Xcode project. The steps are included in the installation instructions.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: AEP-Assurance/Assets/Scenes/AssuranceScene.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public class AssuranceScene : MonoBehaviour
     public static void HandleStartAdobeCallback()
     { 
         print(LOG_TAG + "HandleStartAdobeCallback");
-        ACPCore.ConfigureWithAppID("94f571f308d5/00fc543a60e1/launch-c861fab912f7-development");    
+        ACPCore.ConfigureWithAppID("94f571f308d5/b6b34bf9c29f/launch-7226814f5897-development");
```
Comment (emdobrin) [inline AEP-Assurance/Assets/Scenes/AssuranceScene.cs:31]: We should remove this app id and use an empty constant string:
ACPCore.ConfigureWithAppID(APP_ID);

File: AEP-Assurance/Assets/Plugins/iOS/Editor/BuildPostProcessor.cs [hunk 2]
```diff
@@ -39,6 +38,12 @@ public static void OnPostProcessBuild(BuildTarget target, string path)
     static void AddFrameworks(PBXProject project, string targetGUID)
     {
         project.AddFrameworkToProject(targetGUID, "WebKit.framework", false);
+        project.AddFrameworkToProject(targetGUID, "SystemConfiguration.framework", false);
+        project.AddFrameworkToProject(targetGUID, "UserNotifications.framework", false);
+        project.AddFrameworkToProject(targetGUID, "UIKit.framework", false);
+        project.AddFrameworkToProject(targetGUID, "libsqlite3.tbd", false);
+        project.AddFrameworkToProject(targetGUID, "libz.tbd", false);
+        project.AddFrameworkToProject(targetGUID, "libc++.tbd", false);
```
Comment (emdobrin) [inline AEP-Assurance/Assets/Plugins/iOS/Editor/BuildPostProcessor.cs:46]: Same comment here, we include these in core, they may not be required here

File: README.md [hunk 3]
```diff
@@ -80,54 +92,54 @@ Ensure that Assets/Plugin/Android/AndroidManifest.xml contains the following Int
...
-1. If `Android` is not the active platform, hit the button that says __Switch Platform__ (it will only be available if you actually need to switch active platforms)
-1. Press the __Build And Run__ button
-1. You will be asked to provide a location to save the build. Make a new directory at *unity-aepassurance/AEPAssurance/Builds* (this folder is in the .gitignore file)
-1. Name build whatever you want and press __Save__
-1. `Unity` will build an `apk` file and automatically deploy it to the connected device
+2. From the menu of the `Unity` app, select __File > Build Settings...__
+3. Select `Android` from the __Platform__ window
+4. If `Android` is not the active platform, hit the button that says __Switch Platform__ (it will only be available if you actually need to switch active platforms)
+5. Press the __Build And Run__ button
+6. You will be asked to provide a location to save the build. Make a new directory at *unity-aepassurance/AEP-Assurance/Builds* (this folder is in the .gitignore file)
```
Comment (emdobrin) [inline README.md:120]: is this correct or should it be AEPAssurance

File: AEP-Assurance/Assets/Scenes/Tests/AssuranceTestsScript.cs [hunk 4]
```diff
@@ -26,7 +26,7 @@ public IEnumerator VerifyAssuranceVersion()
 			var button = gameObj.GetComponent<Button>();
 			button.onClick.Invoke();
 			yield return new WaitForSeconds(1f);
-			Assert.AreEqual("Assurance Version: "+ (Application.platform == RuntimePlatform.Android ? "1.1.7" : "1.1.0"), getActualResult());
+			Assert.AreEqual("Assurance Version: "+ (Application.platform == RuntimePlatform.Android ? "1.0.0" : "1.1.3"), getActualResult());
```
Comment (emdobrin) [inline AEP-Assurance/Assets/Scenes/Tests/AssuranceTestsScript.cs:29]: Can we update android to latest?

File: README.md (hunk 3 above)
Comment (spoorthipujariadobe) [reply README.md:120]: The folder was named AEP-Assurance for this one

File: AEP-Assurance/Assets/Scenes/Tests/AssuranceTestsScript.cs (hunk 4 above)
Comment (spoorthipujariadobe) [reply AEP-Assurance/Assets/Scenes/Tests/AssuranceTestsScript.cs:29]: The build fails with 1.0.3 for some reason, I can investigate a bit more if we absolutely want to do it.

Comment (emdobrin) [reply AEP-Assurance/Assets/Scenes/Tests/AssuranceTestsScript.cs:29]: Can we log a task for this so it does not get lost?

File: README.md (hunk 3 above)
Comment (emdobrin) [reply README.md:120]: Let's rename to AEPAssurance for consistency

**Review by emdobrin — COMMENTED** [review]
Looks good with some renaming work

File: AEP-Assurance/Assets/Scenes/Tests/AssuranceTestsScript.cs (hunk 4 above)
Comment (spoorthipujariadobe) [reply AEP-Assurance/Assets/Scenes/Tests/AssuranceTestsScript.cs:29]: https://jira.corp.adobe.com/browse/MOB-15722

**Pull request merged**